cirq-rigetti/cirq_rigetti/quil_input_test.py

```diff
@@ -122,8 +139,10 @@ def test_circuit_from_quil():
         ]
     )
     # build the same Circuit, using Quil
-    quil_circuit = circuit_from_quil(QUIL_PROGRAM)
+    quil_circuit = circuit_from_quil(Program(QUIL_PROGRAM))
     # test Circuit equivalence
+    print(cirq_circuit)
+    print(quil_circuit)
```

Comment:
Looks like a debug statement, please remove.  Also please check in other test modules, e.g.,
```
git diff -S print main...HEAD -- "*test.py" | egrep '^[+]\s*print'
```

Reply:
Removed the once present in `cirq-rigetti`